fields/types/date/DateFilter.js

```diff
@@ -17,16 +17,16 @@ const MODE_OPTIONS = [
   { label: 'Between', value: 'between' },
 ];
 
-var DayPickerIndicator = React.createClass({
-  render() {
-    return (
-      <span className="DayPicker-Indicator">
-        <span className="DayPicker-Indicator__border" />
-        <span className="DayPicker-Indicator__bg" />
-      </span>
-    );
-  },
-});
+const DayPickerIndicator = ({ activeInputField }) => {
```

Comment:
Should we add props type defined here ?

Reply:
Updated in https://github.com/twreporter/keystone/pull/197/commits/24c4391afb99671a55a81821d85e71ff942a1ec7.